### PR TITLE
yarn.lock security update

### PR DESCRIPTION
**Pull request by fatihcandev**
See https://github.com/DIGIEGGS/rn-country-code-picker/security/dependabot/yarn.lock/node-notifier/open

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -5832,8 +5832,8 @@ node-modules-regexp@^1.0.0:
   resolved "https://registry.yarnpkg.com/node-modules-regexp/-/node-modules-regexp-1.0.0.tgz#8d9dbe28964a4ac5712e9131642107c71e90ec40"
   integrity sha1-jZ2+KJZKSsVxLpExZCEHxx6Q7EA=
 
-node-notifier@^6.0.0:
-  version "6.0.0"
+node-notifier@^8.0.1:
```
Comment (isatanis) [inline yarn.lock:5835]: We should find which library is using this dependency and we will upgrade it if it is possible. It will be more consistent.

**Review by isatanis — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 1 above)
Comment (fatihcandev) [reply yarn.lock:5835]: I've searched the project but node-notifier hasn't showed up in the node_modules.

**Comment by fatihcandev** [comment]
I ran a `yarn audit` command and it showed me this vulnerability. It seems we don't have any other choice rather than updating yarn.lock.

image

**Review by isatanis — APPROVED** [review]
It's related to the `Jest` package. We should upgrade it when Jest version 27.0.0 is released.

**Pull request merged**